app/javascript/components/FacetControl.js

```diff
@@ -51,13 +68,17 @@ export default function FacetControl(props) {
     };
   }, []);
 
+  let controlContent = facetName
+  if (selectedFilterString) {
+    controlContent = <>{ selectedFilterString } <button ref={clearNode} className="facet-clear" onClick={ clearFacet }><FontAwesomeIcon icon={faTimes}/></button></>
```

Comment:
I made the same comment on another review:

> I'm not a fan of reassigning variables. I personally avoid them. I think it can introduce problems such as:

> 1. You can change the external state on accident. Since it defined in the global context it's possible for another developer to reassign the value inside of a function if they use the same name. Or it's possible to repurpose the use of the variable on accident. This may cause unexpected behavior. 
> 2. It can make the code more complex. When we see 'let' I know the value is going to change. But it's hard to keep track of. 
> 
> I think if there could be a way to decide the final value of the variable it would make things easier. But the choice is up to you! And I misunderstood this code, please disregard.

Reply:
Yeah, I agree that in React, reassigning is something that should be done with great care.  I actually tried using a ternary operator with this logic at first, but couldn't get the jsx syntax quite right.  I'll take another stab at it